compiler/noirc_evaluator/src/ssa/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -2128,7 +2127,11 @@ fn execute_brillig<F: AcirField>(
     }
 
     // Instantiate a Brillig VM given the solved input registers and memory, along with the Brillig bytecode.
-    let mut vm = VM::new(calldata, code, Vec::new(), &blackbox_solver::StubbedBlackBoxSolver);
+    //
+    // We pass a stubbed solver here as we a concrete solver implies a field choice which conflicts with this function
+    // being generic.
+    let solver = acvm::blackbox_solver::StubbedBlackBoxSolver;
```

Comment:
I added a comment above to explain this (although it's a little broken). The function is generic over the field whereas `Bn254BlackBoxSolver` isn't so we'd need to pass in the solver as an argument rather than construct it within the function. I took the view that we can just ignore this for now and improve later rather than restructure things in this PR.

Reply:
Ah, I misread the comment